ow_plexil/src/plans/ArmMoveJoint.plp

```diff
@@ -0,0 +1,37 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// A plan for the ROS action ArmMoveJoint
+// that moves a single arm joint
+
+#include "plan-interface.h"
+
+ArmMoveJoint:
+{
+  In Boolean Relative;
+  In Integer Joint;
```

Comment:
Would be good to check that Joint is in the range 0-5, and issue an error otherwise.  Once we learn the range of each joint, those could be checked here as well.